src/views/viewer/ChannelView/ChannelView.tsx

```diff
@@ -71,18 +99,25 @@ export const ChannelView: React.FC = () => {
     try {
       if (isFollowing) {
         openUnfollowDialog({
-          variant: 'info',
+          variant: 'error',
           exitButton: false,
           error: true,
-          description: `Do you want to unfollow ${channel?.title}?`,
+          title: 'Would you consider staying?',
+          description: (
+            <>
+              {channel?.title} released <DialogAccentText>{videosLastMonth} new videos </DialogAccentText>
+              this month.
+              <br /> Cancel to follow for more fresh content!
+            </>
+          ),
```

Comment:
Where does this come from? I don't see this in the design

Reply:
It's spread over several files sadly https://www.figma.com/file/TGUpBXgXf9ceMvHbRB5BIx/Joystream-Atlas?node-id=10582%3A289758

Reply:
@KubaMikolajczyk let's try to keep prototype consistent with the design 🙏

Reply:
Also I'm a bit unsure about this message. The point of the dialog is to make sure user doesn't unfollow by accident. This looks more like trying to keep the user to follow. If they didn't click the button by accident why would we try to persuade them to not unfollow? I personally don't like this type of communication from the app and I think it should be used only if the user wants to stop paying us or something, if ever.

Reply:
@kdembler I promise to make it consistent - where is it inconsistent? 

Firstly - there was a wording mistake on my end "keep following for.." instead "cancel to follow". sorry :)

If it comes to the dialog itself - I hear you - we could only ask if the user wants to unfollow and thats it. The reason I wanted to introduce this pattern was the fact that currently each engaged user is really valuable - when user decides to unfollow - both viewer and publisher loses some added value to their experience with the platform.

Please notice that we don't use any dark pattern here like switching the primary action to keep following and confusing user - just asking question and showing some additional information that may be helpful for the viewer.

Maybe @bedeho could add something to this discussion?

I made a second version also: [Link](https://www.figma.com/file/TGUpBXgXf9ceMvHbRB5BIx/Joystream-Atlas?node-id=11921%3A424903)

![image](https://user-images.githubusercontent.com/31144205/127155653-4901620f-e614-4c54-9b6b-9379fa533985.png)

Reply:
Regarding inconsistency, [here](https://www.figma.com/file/TGUpBXgXf9ceMvHbRB5BIx/Joystream-Atlas?node-id=8171%3A210739) a different dialog is used.

I like the second one more, but I see your point as well. I think we will need a tiebreaker @bedeho :D